### PR TITLE
Community Article - Using Holland and Cloud Backup to Backup Database…

### PR DESCRIPTION
**Pull request by BadCrumpany**
…s with Ubuntu

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-99fcd1e5bc/how-to/)

**Comment by stephamon** [comment]
Hey @cyrichardson can you take a look at this one please?

**Comment by cyrichardson** [comment]
@stephamon On it!

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-8599f68bfd/how-to/). :bow:

**Comment by cyrichardson** [comment]
@stephamon Suggested changes here: 
[using-holland-and-cloud-backup-to-backup-databases-with-Ubuntu_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2813640/using-holland-and-cloud-backup-to-backup-databases-with-Ubuntu_cyrEdits.docx)

File: content/cloud-servers/using-holland-and-cloud-backup-to-backup-databases-with-Ubuntu.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+permalink: using-holland-and-cloud-backup-to-backup-databases-with-Ubuntu/
```
Comment (RackerKate) [inline content/cloud-servers/using-holland-and-cloud-backup-to-backup-databases-with-Ubuntu.md]: Should the "U" be lowercase in the permalink?

Also, suggest changing the permalink and file name to "Use..."

File: content/cloud-servers/using-holland-and-cloud-backup-to-backup-databases-with-Ubuntu.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+If you don't have that file, create one and enter the following information:
+
+     [client]
+
+      user=root
+
+      password=yourpasswordhere
+
+The default location for your backups is **/var/spool/holland**. If you want to store your backups in a different
+location, open the **/etc/holland/holland.conf** file in a text editor, and edit the following line:
```
Comment (RackerKate) [inline content/cloud-servers/using-holland-and-cloud-backup-to-backup-databases-with-Ubuntu.md]: Suggest removing the 2nd comma.

File: content/cloud-servers/using-holland-and-cloud-backup-to-backup-databases-with-Ubuntu.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+    backup_directory = /var/spool/holland
+
+Holland stores only one backup in the backup directory by default. If you want to have multiple backups in your backup
+directory, open the **/etc/holland/backupsets/default** file in a text editor, and edit the following line:
```
Comment (RackerKate) [inline content/cloud-servers/using-holland-and-cloud-backup-to-backup-databases-with-Ubuntu.md]: Same as above.

File: content/cloud-servers/using-holland-and-cloud-backup-to-backup-databases-with-Ubuntu.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+      lrwxrwxrwx 1 root root   42 Sep 29 19:37 oldest -> /var/spool/holland/default/20130929_193720
+
+### Schedule Holland backups
+
+You can use Holland to schedule regular backups of your database. To do so, open the **/etc/crontab** file and add a new
+cron job. The following example tells Holland to run a backup every day at 3:00 am: 
+
+    0 3 * * * root holland bk
+
+The following syntax is for forming a cron job:
```
Comment (RackerKate) [inline content/cloud-servers/using-holland-and-cloud-backup-to-backup-databases-with-Ubuntu.md]: Suggest "The following syntax forms a cron job:"

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d7183e9262/how-to/).

**Pull request merged**